src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => resolve());
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
+});
+
+const successHandler = () => {
+  const successMessage = document.createElement('div');
+
+  successMessage.className = 'message';
+  successMessage.textContent = 'Promise was resolved!';
+  document.body.appendChild(successMessage);
+};
+
+const errorHandler = (error) => {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.className = 'message error-message';
+  errorMessage.textContent = error.message;
+  document.body.appendChild(errorMessage);
+};
+
+promise1.then(successHandler).catch(errorHandler);
+promise2.then(successHandler).catch(errorHandler);
```

Comment:
Currently, both promises are handled separately. If you want to handle them together, consider using `Promise.all` or `Promise.race` depending on the desired behavior. This will allow you to handle the completion of both promises in a more coordinated manner.